packages/salesforcedx-utils-vscode/test/jest/helpers/paths.test.ts

```diff
@@ -70,4 +73,44 @@ describe('test project paths', () => {
       expect(relativeStateFolder).toEqual(FAKE_STATE_FOLDER);
     });
   });
+
+  describe('test SFDXGlobalConfigPath', () => {
+    it('should return the global config path', () => {
+      const mockOS = jest.requireActual('os');
+      mockOS.homedir = jest.fn().mockReturnValue(FAKE_HOME_DIR);
+      jest.mock('os', () => mockOS);
+      const result = projectPaths.sfdxGlobalConfig();
+      expect(result).toEqual(path.join(FAKE_HOME_DIR, FAKE_STATE_FOLDER, FAKE_CONFIG_FILE)
+      );
+    });
+  });
+
+  describe('test SFDXConfigPath', () => {
+
+    it('should be defined', () => {
+      expect(projectPaths.sfdxConfig).toBeDefined();
+    });
+
+    it('should return a path to the project config file', () => {
+      const FAKE_CONFIG = path.join(FAKE_WORKSPACE, FAKE_STATE_FOLDER, FAKE_CONFIG_FILE);
+      const FAKE_PATH = path.join(FAKE_WORKSPACE, FAKE_STATE_FOLDER);
```

Comment:
What do you think about moving these consts to the top of the file so they can be reused in the expect statements?

Reply:
I think the first one is something specific to this it so I'd leave it there, but the second one I'm going to move it to the top because there's another it using the same path:)